ToolBox/GDAXDownloader/GDAXDownloader.cs

```diff
@@ -12,26 +12,32 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
 */
-using System;
-using System.Collections.Generic;
-using System.Net;
-using QuantConnect.Data;
-using QuantConnect.Data.Market;
-using Newtonsoft.Json;
-using System.Linq;
-using System.Text;
-using Newtonsoft.Json.Linq;
-using QuantConnect.Logging;
 
 namespace QuantConnect.ToolBox.GDAXDownloader
 {
+    using System;

```

Comment:
Normally we've kept the using statements outside the namespace. Shouldn't be needed here.